core/geometry/src/geometry3d/AngleSweep.ts

```diff
@@ -184,12 +184,22 @@ export class AngleSweep implements BeJSONFunctions {
     else
       return AngleSweep.createStartEndRadians(this.endRadians, this.startRadians + s * Math.PI, result);
   }
-  /** Restrict start and end angles into the range (-90,+90) in degrees */
-  public capLatitudeInPlace() {
+  /** Restrict start and end angles into the range (-90,+90) in degrees. */
+  public capLatitudeInPlace(): void {
     const limit = 0.5 * Math.PI;
     this._radians0 = Geometry.clampToStartEnd(this._radians0, -limit, limit);
     this._radians1 = Geometry.clampToStartEnd(this._radians1, -limit, limit);
   }
+  /**
+   * Restrict angle sweep into the range (-360,+360) in degrees.
+   * Start angle is unchanged and sign of sweep is unchanged.
```

Comment:
It's a closed interval, plus wordsmithing:

```suggestion
   * Restrict the sweep angle to the degree range [-360, 360].
   * The start angle and sign of the sweep angle are unchanged.
```